packages/api/src/util/customDecorators.ts

```diff
@@ -23,3 +32,12 @@ function decorateExtrinsic(
     };
     return (api as any).decorateFunctionMeta(method, newMethod);
 }
+
+function decorateExtrinsicsSubmittable(
```

Comment:
this method can be combined with decorateExtrinsic.
replace `const extrinsic = method(...params);` with `const extrinsic = creator(method(...params));`